# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: third_party/rust/wgpu-hal/Cargo.toml
# Commit: b3109ad14685
# Full Hash: b3109ad14685624ada7861083ff5332aec9a7818
# Author: Andy Leiserson <aleiserson@mozilla.com>
# Date: 2025-10-07 17:00 +0000
# Description:
#   Bug 1992842 - Update wgpu to upstream 1072b878 (2025-10-06) r=webgpu-reviewers,supply-chain-reviewers,ErichDonGubler
# ==============================================================================

diff -r 1d3f000cf1f9 -r b3109ad14685 third_party/rust/wgpu-hal/Cargo.toml
--- a/third_party/rust/wgpu-hal/Cargo.toml	Tue Oct 07 16:51:44 2025 +0000
+++ b/third_party/rust/wgpu-hal/Cargo.toml	Tue Oct 07 17:00:40 2025 +0000
@@ -13,7 +13,7 @@
 edition = "2021"
 rust-version = "1.82.0"
 name = "wgpu-hal"
-version = "26.0.0"
+version = "27.0.0"
 authors = ["gfx-rs developers"]
 build = "build.rs"
 autolib = false
@@ -203,7 +203,7 @@
 optional = true
 
 [dependencies.naga]
-version = "26.0.0"
+version = "27.0.0"
 path = "../naga"
 
 [dependencies.ordered-float]
@@ -234,7 +234,7 @@
 default-features = false
 
 [dependencies.wgpu-types]
-version = "26.0.0"
+version = "27.0.0"
 path = "../wgpu-types"
 default-features = false
 
@@ -246,7 +246,7 @@
 version = "0.30.7"
 
 [dev-dependencies.naga]
-version = "26.0.0"
+version = "27.0.0"
 path = "../naga"
 features = [
     "wgsl-in",